src/views/app/detail-panel/edit/parts/test/change-signatures-dropdown.test.tsx

```diff
@@ -0,0 +1,107 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import React from 'react';
+
+import { act, screen } from '@testing-library/react';
+import { noop, times } from 'lodash';
+import { HttpResponse } from 'msw';
+
+import { createAPIInterceptor } from '../../../../../../carbonio-ui-commons/test/mocks/network/msw/create-api-interceptor';
+import { getEmptyMSWShareInfoResponse } from '../../../../../../carbonio-ui-commons/test/mocks/network/msw/handle-get-share-info';
+import { setupTest } from '../../../../../../carbonio-ui-commons/test/test-setup';
+import { addEditor } from '../../../../../../store/zustand/editor';
+import { generateNewMessageEditor } from '../../../../../../store/zustand/editor/editor-generators';
+import { setupEditorStore } from '../../../../../../tests/generators/editor-store';
+import { buildSignature } from '../../../../../../tests/generators/signatures';
+import { generateStore } from '../../../../../../tests/generators/store';
+import { handleGetSignaturesRequest } from '../../../../../../tests/mocks/network/msw/handle-get-signatures';
+import { SignItemType } from '../../../../../../types';
+import { EditView, EditViewProp } from '../../edit-view';
+
+describe('Change signature while composing mail', () => {
+	/**
+	 * Test the creation of a new email with change signature option
+	 */
+	it('Change singnatures icon should show if user have signatures', async () => {
+		createAPIInterceptor(
+			'post',
+			'/service/soap/GetShareInfoRequest',
+			HttpResponse.json(getEmptyMSWShareInfoResponse())
+		);
```

Comment:
```suggestion
		createSoapAPIInterceptor('GetShareInfo')
```
I suggest the usage of this function. Is more compact and it relieves you from the need to provide a dummy response, the HTTP method and the full endpoint URL